spock-core/src/main/java/org/spockframework/runtime/model/SpecInfo.java

```diff
@@ -126,6 +126,16 @@ public boolean getIsBottomSpec() {
     return subSpec == null;
   }
 
+  public List<SpecInfo> getSpecsToBottom() {
+    List<SpecInfo> topToToBottom = getSpecsTopToBottom();
+    return topToToBottom.subList(topToToBottom.indexOf(this), topToToBottom.size());
```

Comment:
Why so verbose? Actually, shortening loops like that into using `subList` is even a suggested IDEA refactoring. Or is your concern rather some kind of premature optimisation because you want to avoid walking the whole chain from top to bottom?